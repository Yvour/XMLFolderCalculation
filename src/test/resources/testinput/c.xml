
<expressions>

	<addition id="190" complex="true">
		<item>2</item>
		<item>3</item>
		<item>
			<subtraction>
				<minuend>7</minuend>
				<subtrahend>
					<addition>
						<item>1</item>
						<item>1</item>
					</addition>
				</subtrahend>
			</subtraction>
		</item>
	</addition>
</expressions>