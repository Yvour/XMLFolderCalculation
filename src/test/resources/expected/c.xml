<?xml version="1.0" ?><expressions><result id="190">10</result></expressions>